detection_rules/cli_utils.py

```diff
@@ -113,7 +113,7 @@ def rule_prompt(path=None, rule_type=None, required_only=True, save=True, verbos
         while True:
             try:
                 contents['query'] = click.edit(contents['query'], extension='.eql')
-                rule = Rule(path, {'rule': contents})
+                rule = TOMLRule(path, {'rule': contents})
```

Comment:
Should L130 be changed to `rule.save_toml()` or `rule.save_json()`?

Reply:
fixed in 43218e49c9b0aa1342b21783d451a3ee1c49d3e9